src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -23,6 +23,7 @@ public class PreprocessPackageDependenciesDesignTime : TaskBase
         public const string DependenciesMetadata = "Dependencies";
         public const string CompileTimeAssemblyMetadata = "CompileTimeAssembly";
         public const string ResolvedMetadata = "Resolved";
+        public const string VisibleMetadata = "Visible";
```

Comment:
Just to be absolutely clear: this is the metadata we want to use to control whether or not an item is visible by default in Solution Explorer, right>